plugin.video.videomediaset/resources/settings.xml

```diff
@@ -4,6 +4,7 @@
         <setting label="32002" type="text" id="password" option="hidden" default=""/>
         <setting label="32003" type="number" id="itemsperpage" default="500"/>
         <setting label="32004" type="bool" id="fullguide" default="false"/>
+        <setting label="32007" type="bool" id="sortmediaset" default="false"/>
```

Comment:
it'd be nice if you can convert the settings to the new format at some point

Reply:
Right i forgot about that. If it's ok I'll update it now for the next release and I don't update this one..your call